vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/json/JnKeysScalarEvaluator.java

```diff
@@ -66,9 +68,10 @@ protected void evaluate(TaggedValuePointable[] args, IPointable result) throws S
                 for (int i = 0; i < size1; i++) {
                     sp1.getEntry(i, temptvp);
                     if (temptvp.getTag() == ValueTag.OBJECT_TAG) {
-                        op = (ObjectPointable) ObjectPointable.FACTORY.createPointable();
                         temptvp.getValue(op);
-                        op.getKeys(temptvp);
+                        abvs1 = new ArrayBackedValueStorage();
```

Comment:
There's a PointablePooleFactory. Am I to create an  ArrayBackedValueStoragesPoolFactory?